docs/ai/conceptual/embeddings.md

```diff
@@ -0,0 +1,75 @@
+
+## Embeddings preserve semantic meaning
+
+Embeddings turn non-numeric data into numeric data that represents semantic meaning. Specifically, an embedding is a vector&mdash;an array of numeric values&mdash; that an LLM uses to compare semantic meanings. Two embeddings with similar vectors also have similar semantic meanings, such as an image and its alt text, or a course syllabus and the textbook for that course.
+
+### Choosing embedding models to generate embeddings
+
+You generate embeddings for your raw data by using an embedding model: an AI model that can encode a piece of non-numeric data into a vector (a long array of numbers). The model can also decode an embedding into non-numeric data that has the same or similar meaning as the original, raw data.
+
+In Semantic Kernel, you use connectors to integrate embedding models and databases into your kernel, so you can quickly pull in text data, and generate and store embeddings. This lets you use a vector database solution to store and retrieve semantic memories.
+
+In Azure OpenAI, the service includes access to [embedding models](/azure/ai-services/openai/concepts/models#embeddings).
+
+### Store and process embeddings in a vector database
+
+Vector databases are designed to store and process vectors, so they're a natural home for embeddings. Different vector databases offer different processing capabilities, so you should choose one based on your raw data and your goals. For information about your options, see [available vector database solutions](vector-databases.md#available-vector-database-solutions).
+
+## Use cases for embeddings
+
+This section lists the main use cases for embeddings. LLMs can process text embeddings, but to use other data types you need to include a suitable embedding model, such as **dall-e-3** for image generation.
+
+> [!TIP]
+> In Semantic Kernel, you can [use nested functions](/semantic-kernel/prompts/calling-nested-functions?tabs=Csharp#calling-a-nested-function) to integrate embeddings in your prompts.
+
+### Use your own data to improve completion relevance
+
+Use your own databases to generate embeddings for your data and integrate it with an LLM to make it available for completions. This use of embeddings is called [retrieval-augmented generation (RAG)](/).
```

Comment:
Should this link be added?

Reply:
The target isn't ready yet. Removing for now...